vscode/oso/syntaxes/polar.tmLanguage.json

```
@@ -139,7 +139,13 @@
       "captures": {
         "2": { "name": "constant.character" }
       }
+    },
+    "operator": {
+      "match": "(+|-|*|/|<|>|=|!)",
```

```suggestion
      "match": "(+|-|*|/|<|>|=|!)=?",
```
does this work for matching `==`, `!=`, `>=`, `<=`?

---

You don't need that, the regex already matches `<` and `=` so it's correctly hilighted.